src/kernels/jupyter/launcher/liveshare/hostJupyterServer.ts

```diff
@@ -34,27 +34,35 @@ import { JupyterNotebook } from '../jupyterNotebook';
 import { noop } from '../../../../platform/common/utils/misc';
 import { Cancellation } from '../../../../platform/common/cancellation';
 import { getDisplayPath } from '../../../../platform/common/platform/fs-paths';
-import { INotebookServer, IJupyterSessionManagerFactory } from '../../types';
+import { INotebookServer } from '../../types';
 import { Uri } from 'vscode';
 /* eslint-disable @typescript-eslint/no-explicit-any */
 
 @injectable()
```

Comment:
You should probably remove this now then. Otherwise I think the tests will fail saying it's not in the container.